package.json

```diff
@@ -15,25 +15,26 @@
     "upgrade:interactive": "yarn upgrade-interactive --latest"
   },
   "dependencies": {
-    "@reduxjs/toolkit": "1.6.2",
+    "@entitree/helper": "0.0.2",
+    "@reduxjs/toolkit": "1.8.0",
     "@types/axios": "0.14.0",
-    "@types/luxon": "2.0.5",
+    "@types/luxon": "2.3.0",
     "@types/wikidata-sdk": "6.1.0",
-    "axios": "0.24.0",
+    "axios": "0.26.1",
     "babel-plugin-macros": "3.1.0",
-    "bootstrap": "4.6.0",
-    "cookie": "0.4.1",
-    "entitree-flex": "0.4.0",
+    "bootstrap": "5.1.3",
```

Comment:
uh watch out, I think I did it once and some styles screwed up, I think it was the header

Reply:
ok let's check